### DISCUSSION TITLE
Colors of 256 palette are not propriate on example

### DISCUSSION THREAD

**Discussion by ZalgoSoft** [magiblot/tvision#93] [General]
256 color palette means at least 15 shades of red, like this 

![изображение](https://user-images.githubusercontent.com/80800403/215063341-f0da57ed-481f-44ce-906d-5611b06ba95c.png)

or more correct 

![изображение](https://user-images.githubusercontent.com/80800403/215067631-2ca27376-9b76-4f10-8956-1101b0f9f413.png)

but your example contains like this 

![изображение](https://user-images.githubusercontent.com/80800403/215063461-e399442f-7782-4d9f-bf7b-c0a0389a15e7.png)

**Comment by magiblot** [comment]
Hi @ZalgoSoft, thanks for the comment.

I guess that depends on which 256-color palette we are talking about.

In the case of Turbo Vision we are not talking of an RGB-based 256-color palette, but of an Xterm-based 256-color palette, which consists of (https://www.ditig.com/256-colors-cheat-sheet):

* 16 default terminal colors.
* A 6x6x6 color cube (216 colors).
* 24 grayscale tones.

The example in the README depicts how a 24-bit color picture would be shown in a terminal using the `xterm-256color` palette, and I think the result is quite good.

**Comment by jcarp-7ab** [comment]
Is this 256 color issue the reason my example looks so strange??

![Screenshot 2024-11-02 9 31 49 PM](https://github.com/user-attachments/assets/a4611b31-a2e0-4ca1-8b81-41d18ec9e91e)

**Reply by jcarp-7ab** [reply]
I am such a goof...  I need to learn about my own computer.  It's a Chromebook with a Debian Linux guest... and my Color issue is that the Terminal has multiple color themes!  So I needed to set the colors or change the theme.  #EMBARRASSED

![Screenshot 2024-11-03 12 42 13 AM](https://github.com/user-attachments/assets/4fbcbd85-4fb8-464b-8f48-a7e4afba3951)

![Screenshot 2024-11-03 1 00 22 AM](https://github.com/user-attachments/assets/982c4315-9304-4faf-9953-fcad6a72143c)

I have so much to learn, yet I was able to get this stuff running and the colors fixed...Progress
